<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelistedTool</class>
 <widget class="QMainWindow" name="DelistedTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1174</width>
    <height>681</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1174</width>
    <height>681</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DelistedTool_V1.0.1_1906241925</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico.qrc">
    <normaloff>:/ico1/退板记录.png</normaloff>:/ico1/退板记录.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>248</y>
      <width>1151</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>退板查询</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>24</y>
       <width>1131</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>时间段：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2019</year>
            <month>6</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 75 9pt &quot;ADMUI3Lg&quot;;</string>
          </property>
          <property name="text">
           <string>----&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_2">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2019</year>
            <month>6</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>ID：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="IDlineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>订单号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zhidanlineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePart">
          <property name="text">
           <string>删除选中数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAll">
          <property name="text">
           <string>删除所有数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="writrToExcel">
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="DelistedCheck">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/*tablewidget 样式*/
QTableView {
    selection-background-color:rgb(210,246,251);
	selection-color:black;
    background-color:white;     /*整个表格的背景色，这里为白色*/
    border:1px solid #E0DDDC;   /*边框为1像素，灰色*/
    gridline-color:lightgray;   /*这个是表格的格子线的颜色，为亮灰*/
}
 
/*这里是表格表头样式*/
QHeaderView::section {
    background-color:rgb(238,232,231);     /*背景色 白色*/
    border:0.5px solid #E0DDDC;   /*先把边框宽度设为0，即隐藏所有表头边框*/
    border-bottom:1px solid #E0DDDC;/*然后只显示下边框，因为上边框和左右边框是整个Table的边框，都显示会有2px的边框宽度*/
    height:20px;                /*表头高度*/
}
</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>退板操作</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>订单号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ZhiDanBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;黑体&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>本机IP：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PCAddress">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="IDCtrl">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">
QToolButton#delAlarmBtn{

       font-size: 12px;

       border-style:inset;

       border:0px solid #E5E5E5;

  
    qproperty-iconSize: 16px 16px;//图片大小

    qproperty-toolButtonStyle: ToolButtonTextBesideIcon;//文字和图片样式

}</string>
       </property>
       <property name="text">
        <string>刷新订单</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输入ID号：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>751</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>退板日志</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="delistedLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;</string>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="channgeUser">
   <property name="text">
    <string>切换账号</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="logOut">
   <property name="text">
    <string>退出账号</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
